addons/mail/tests/test_link_preview.py

```diff
@@ -170,3 +170,28 @@ def test_link_preview_no_content_type(self):
             url = self.source_url
             session = requests.Session()
             link_preview.get_link_preview_from_url(url, session)
+
+    def test_link_preview_ignore_list(self):
+        with patch.object(requests.Session, 'get', self._patch_with_og_properties), patch.object(requests.Session, 'head', self._patch_head_html):
+            message = self.test_partner.message_post(
+                body=Markup(f'<a href="{self.source_url}">Nothing link</a>'),
+            )
+            self.env["mail.link.preview"]._create_from_message_and_notify(message)
+            link_preview_count = self.env["mail.link.preview"].search_count([("message_id", "=", message.id)])
+            self.assertEqual(link_preview_count, 1)
+
+            self.env["ir.config_parameter"].sudo().set_param("mail.mail_link_preview_ignore_list", self.source_url)
+            message = self.test_partner.message_post(
+                body=Markup(f'<a href="{self.source_url}">Nothing link</a>'),
+            )
+            self.env["mail.link.preview"]._create_from_message_and_notify(message)
+            link_preview_count = self.env["mail.link.preview"].search_count([("message_id", "=", message.id)])
+            self.assertEqual(link_preview_count, 0)
```

Comment:
The goal of having a single test would be to not repeat the code, but instead loop on (url, expected result) (or at least call a method doing all the processing)